src/uct/base/uct_iface.c

```diff
@@ -259,6 +259,21 @@ ucs_status_t uct_base_iface_fence(uct_iface_h tl_iface, unsigned flags)
     return UCS_OK;
 }
 
+ucs_status_t uct_base_iface_get_max_conn_priv(uct_iface_h tl_iface, size_t *max_conn_priv)
```

Comment:
why do we need this function in common code? if it's rdmacm, it can tell its own max_conn_priv. for API, we have the query.